docs/conceptual/list.comparewith['t]-function-[fsharp].md

```diff
@@ -0,0 +1,71 @@
+---
+title: List.compareWith<'T> Function (F#)
+description: List.compareWith<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
```

Comment:
Could you make sure that you add the full metadata table as in other documentation pieces?